app/src/main/res/layout/recyclerview_row.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+xmlns:android="http://schemas.android.com/apk/res/android"
+android:layout_width="match_parent"
+android:layout_height="wrap_content"
+android:orientation="horizontal"
+android:padding="10dp">
+
+<TextView
```

Comment:
Kindly ensure that this file meets the acceptance criteria below and also ensure a more descriptive IDs.


```Acceptance Criteria:
Given I am a dev designing the layout of the RecyclerView
Then I should ensure that the following components are designed well:

User’s profile image
User’s GitHub username

Reply:
On it